R/inboveg_synonym.R

```diff
@@ -0,0 +1,62 @@
+#' Provide accepted names list from INBOVEG database
+#'
```

Comment:
This could also be implemented in the 'coverage-layer' query? so al the preferred names for species are used instead of the synonyms ... or this can be done by the users themselves?